Add docs to mention glob behavior of trace includes/excludes (#68463)
vercel/next.js: docs/02-app/02-api-reference/05-next-config-js/output.mdx

    },
    outputFileTracingIncludes: {
      '/api/another': ['./necessary-folder/**/*'],
      '/api/login/\\[\\[\\.\\.\\.slug\\]\\]': [
        './node_modules/aws-crt/dist/bin/**/*',
      ],
    },
  },
}
```

> Note: the key for `outputFileTracingIncludes`/`outputFileTracingExcludes` is a [glob](https://www.npmjs.com/package/picomatch#basic-globbing) so needs to have special characters escaped.

Reviewer: ```suggestion
**Note:** The key of `outputFileTracingIncludes`/`outputFileTracingExcludes` is a [glob](https://www.npmjs.com/package/picomatch#basic-globbing), so special characters need to be escaped.
```
